mapdata/WarcraftLegacies/AbilityData/Skin/1297035329.json

```diff
@@ -8,16 +8,22 @@
     {
       "Id": 1718840929,
       "Type": 3,
-      "Value": "(Twilight DK)"
+      "Value": "(Scourge DK)"
     },
     {
       "Id": 2020631137,
-      "Value": 2
+      "Value": 1
     },
     {
       "Id": 2037082209,
       "Type": 3,
-      "Value": "E"
+      "Value": "W"
+    },
+    {
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Raises 3 dead units in an area for 60 seconds."
```

Comment:
This is a vanilla ability so you should use data references in the tooltip, like vanilla does. This one would be something like "Raises <A0OM,DataA1> dead units in an area for <A0OM,Dur1> seconds."

Reply:
does that work on non vanilla abilities too? just out of interest

Reply:
It'll work on any object that's defined entirely through the object editor. If it has any C# code defining its behaviour, we probably need to write out values manually.

Reply:
Changed